ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -651,13 +651,17 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
-- as: vsphere-8-ipi-private-proxy-f14-disasterrecovery
+- as: vsphere-ipi-proxy-f14-disasterrecovery
   cron: 11 1 8,22 * *
   steps:
-    cluster_profile: vsphere-8-vpn
+    cluster_profile: vsphere-dis-2
     env:
+      FIPS_ENABLED: "true"
```

Comment:
The test name does not indicate fips. Perhaps update the test name or remove this line to make them in sync for fips status.

Reply:
updated.